accessibility_monitoring_platform/settings/base.py

```diff
@@ -237,6 +237,6 @@
 
 CSP_DEFAULT_SRC = ("'none'",)
 CSP_STYLE_SRC = ("'self'", "'unsafe-inline'")
-CSP_SCRIPT_SRC = ("'self'", )
-CSP_FONT_SRC = ("'self'")
+CSP_SCRIPT_SRC = ("'self'",)
+CSP_FONT_SRC = "'self'"
```

Comment:
This needs to be in a tuple or list

Reply:
I don't know were there changes came from.

Reply:
They have been (mostly) undone.